src/routes/blogs.js

```diff
@@ -1,17 +1,24 @@
 import express from "express";
-import {param, validationResult} from "express-validator"
-
-import services from "../services"
+import { param, validationResult } from "express-validator";
 
+import services from "../services";
 
 const router = express.Router();
 
-router.get('/', (req, res) => {
-    res.send('We are on blogs');
+router.get("/", (req, res) => {
+  res.send("We are on blogs");
 });
 
-router.post('/', (req, res) => {
-    console.log(req.body);
+router.get(
+  "/:id",
+  param("id")
+    .isLength({ min: 24, max: 24 })
```

Comment:
min and max??? isn't it redundant? 
Isnt there function something like `isLength(24)`?